### ISSUE TITLE
Prosilica as nodelet

### ISSUE THREAD

**Issue by jczarnowski** [ros-drivers/prosilica_driver#9]
Is adding a driver nodelet a planned feature or anyone is already working on it?
I think that it would be viable to have the capability to pass the raw images with zero-copy with image processing nodes.

**Comment by wjwwood** [comment]
I don't believe there is a plan to add this anytime soon, but it would make sense to have a nodelet option.

**Comment by trainman419** [comment]
There's an open pull request for this now, if you're interested in trying it out and providing feedback: #11

**Comment by athackst** [comment]
Yes, please update with feedback.  So far I've only been able to test on prosilica GC2540C and GC2450M.

**Comment by LucidOne** [comment]
I'm having some issues running multiple synchronized cameras at the same time.
It doesn't look like it, but does the nodelet code support a shared semaphore to keep both cameras from reading at the same time?
